docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.2'
+services:
+  server:
+    container_name: "song-server-$VERSION"
+    image: "overture/song-server:$VERSION"
+    build:
+      context: ./
+      dockerfile: Dockerfile.server
+      args:
+        - SONG_VERSION=${VERSION}
+        - SONG_ARTIFACTORY_REPO_NAME=dcc-release
+    network_mode: host
+    environment:
+      SERVER_PORT: 8080
+      AUTH_SERVER_URL: http://localhost:8084/check_token/
+      AUTH_SERVER_CLIENTID: 3kJhz9pNtC0pFHAxr2SPkUkGjXrkWWqGcnPC0vBP
+      AUTH_SERVER_CLIENTSECRET: v9mjRtuEVwpt7cgqnsq6mxtCa5FbUOpKLGh7WX8a1dWbBKfrM3iV3VYMtE60jr3W7GLWtNeYIaJ8EUxPkaInclWVXf64qKdR3IKwyfpDU7JhvWEwIYQYdwV1YAUZjB2e
```

Comment:
are those safe to be here ?

Reply:
yep. These are the ones from Docker for Song:

https://github.com/overture-stack/SONG/blob/develop/song-docker-demo/docker-compose.yml#L51-L53

Its more as a reference